### ISSUE TITLE
New Base Knowledge of BERT

### ISSUE THREAD

**Issue by mderosa001** [mderosa001/Capstone_Issues#9]
### Title
BERT: Pre-training of Deep Bidirectional Transformers for
Language Understanding
### URL
https://arxiv.org/abs/1810.04805
### Summary 
BERT (Bidirectional Encoder Representations from Transformers) is a deep learning model introduced by Devlin et al. (2018) that improves natural language understanding by learning bidirectional contextual representations of words using Transformer encoders. It is pre-trained with two tasks: Masked Language Modeling (MLM), where random words are masked and predicted based on context, and Next Sentence Prediction (NSP), which determines if one sentence follows another. BERT is then fine-tuned on various NLP tasks with minimal modifications. It set new benchmarks on multiple language understanding tasks and has influenced many subsequent NLP models.
### Key Points 
Bidirectional Learning – Unlike previous models, BERT captures context from both left and right directions.
Pre-training Tasks – Uses MLM and NSP to improve contextual word understanding.
Transformer-Based – Built on a multi-layer Transformer encoder architecture.
Fine-Tuning for Tasks – Can be adapted to different NLP tasks with minimal changes.
State-of-the-Art Performance – Achieved top results on GLUE, SQuAD, and other benchmarks.
### Citation
@article{DBLP:journals/corr/abs-1810-04805,
  author       = {Jacob Devlin and
                  Ming{-}Wei Chang and
                  Kenton Lee and
                  Kristina Toutanova},
  title        = {{BERT:} Pre-training of Deep Bidirectional Transformers for Language
                  Understanding},
  journal      = {CoRR},
  volume       = {abs/1810.04805},
  year         = {2018},
  url          = {http://arxiv.org/abs/1810.04805},
  eprinttype    = {arXiv},
  eprint       = {1810.04805},
  timestamp    = {Tue, 30 Oct 2018 20:39:56 +0100},
  biburl       = {https://dblp.org/rec/journals/corr/abs-1810-04805.bib},
  bibsource    = {dblp computer science bibliography, https://dblp.org}
}
